multiversxetl/jobs/load_job_test.py

```diff
@@ -17,6 +18,7 @@ def get_load_path(self, task_pretty_name: str) -> Path:
         return Path(__file__).parent / f"transformed_{task_pretty_name}.json"
 
 
+@pytest.mark.integration
```

Comment:
Temporary workaround, a bit of re-design is needed (decoupling). `bigquery.Client()` raises an exception when host machine isn't authenticated to Google Cloud. Tests can be ran locally.